test/helpers/kubeclient_helper.rb

```diff
@@ -0,0 +1,21 @@
+module KubeclientHelper
+  def kubeclient
+    @kubeclient ||= begin
+      config = Kubeclient::Config.read(ENV["KUBECONFIG"])
+      unless config.contexts.include?("minikube")
+        raise "`minikube` context must be configured in your KUBECONFIG (#{ENV["KUBECONFIG"]})"
+      end
+
+      client = Kubeclient::Client.new(
```

Comment:
Yeah, the default is the current context. [This method](https://github.com/abonas/kubeclient/blob/f489a35027a78ea6f410c76b5449366eadf94e85/lib/kubeclient/config.rb#L35) accepts the context name as an arg, and we should definitely pass it. Good catch.